src/components/Button/ButtonFunction/index.test.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { LayoutType } from '../ButtonContent'
+import { ButtonFunction, clickMe } from '.'
+import renderWithTheme from '../../../testing/renderWithTheme'
+
+test('renders button with function and layout line', () => {
+	const { getByText } = renderWithTheme(
+		<ButtonFunction
+			layout={LayoutType.line}
+			click={clickMe}
```

Comment:
Don't bind function meant for the real application into a test. Use some mock up function instead. :slightly_smiling_face: